pytorch_lightning/plugins/precision/deepspeed_precision.py

```diff
@@ -54,7 +54,9 @@ def backward(
 
         return closure_loss
 
-    def clip_gradients(self, optimizer: Optimizer, clip_val: Union[int, float], norm_type: float = float(2.0)):
+    def clip_gradients(
+        self, model: Any, optimizer: Optimizer, clip_val: Union[int, float], norm_type: float = float(2.0)
```

Comment:
what is `float(2.0)` for?

```suggestion
        self, model: Any, optimizer: Optimizer, clip_val: Union[int, float], norm_type: float = 2.0
```